owners/package.json

```diff
@@ -19,6 +19,7 @@
     "lodash": "4.17.15",
     "minimatch": "3.0.4",
     "probot": "7.5.0",
+    "server": "^1.0.19",
```

Comment:
Use exact versions (without the `^`).

For reference, see this [check](https://github.com/ampproject/amphtml/blob/master/build-system/tasks/check-exact-versions.js) in `amphtml`.

Reply:
As a side-note, renovate-bot auto-fixes that in a followup PR, so it's not a disaster if we miss this :)

Reply:
Well I removed it so TAKE THAT RENOVATE-BOT